src/mono/mono/mini/mini-arm64.c

```diff
@@ -3665,6 +3665,8 @@ mono_arch_output_basic_block (MonoCompile *cfg, MonoBasicBlock *bb)
 			}
 			break;
 		}
+		case OP_XCAST:
```

Comment:
It's worth noting that `As` isn't really a regular cast. It's a reinterpretation of bits and in RyuJIT there is no node created for it. It's purely an API to satisfy the higher level type systems and for the JIT we completely elide it in importation to guarantee it is zero cost.